mobileChatExamples/connectReactNativeChat/src/components/ChatSession.js

```diff
@@ -430,61 +377,56 @@ class ChatSession {
     if (this.nextToken) {
       args.nextToken = this.nextToken
     }
+
     return this.client
       .getTranscript(args)
       .then((response) => {
         const incomingDataList = response.data.Transcript
         this.nextToken = response.data.NextToken
-        const transcriptItems = incomingDataList.map((data) => {
-          const transcriptItem = modelUtils.createItemFromIncoming(data, this.thisParticipant)
-          return transcriptItem
-        })
+
+        const transcriptItems = incomingDataList.map((data) =>
+          modelUtils.createItemFromIncoming(data, this.thisParticipant)
+        )
+
         this._addItemsToTranscript(transcriptItems)
         return transcriptItems
       })
       .catch((err) => {
-        console.log('CustomerUI', 'ChatSession', 'transcript fetch error: ', err)
+        console.log('Transcript fetch error:', err)
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Log Injection occurs when untrusted user input is directly written to log files without proper sanitization. This can allow attackers to manipulate log entries, potentially leading to security issues like log forging or cross-site scripting. To prevent this, always sanitize user input using `encodeURIComponent()` or `DOMPurify.sanitize()` before logging.  Learn more - https://cwe.mitre.org/data/definitions/117.html